src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/missingjavadocmethod/InputMissingJavadocMethodNoJavadoc2.java

```diff
@@ -0,0 +1,123 @@
+package com.puppycrawl.tools.checkstyle.checks.javadoc.missingjavadocmethod;
+
+/* Config:
+ * scope = Scope.PROTECTED.getName()
```

Comment:
Why are some comments/ check properties shown as methods invocations? I would stick with `scope = "protected"`.

Reply:
This is because there are tests which are identical, apart from how the scope property is defined in them. Some use the keyword as is (`public`) whereas some use this method invocation (`Scope.PUBLIC.getName()`). 

>tests which are identical 

These are otherwise duplicate tests, which would have to be removed.

Reply:
If we have more tests that only differ by a string literal used in a property, and a method that returns the same value, please remove them.  The runtime does not know the difference between "public" and `Scope.PUBLIC.getName()` once compiled. CI will show if we needed these tests for some reason.

Reply:
Yeah, I was just a little unsure, since the test names differred by name `testX` vs `testXWithResolver`. I have removeed the duplicate now.

Reply:
Resolved.